bin/helm-deploy

```diff
@@ -7,6 +7,16 @@ set -eo pipefail
 
 HELM_DEFAULT_TIMEOUT=300
 
+format_multiple_values_files() {
+  values_files="$1"
+  formatted_files=""
+  echo "${values_files}" | sed -n 1'p' | tr ',' '\n' | ( while read file_name; do
+    file="./deploy/${file_name}.values.yml,"
+    formatted_files="$formatted_files$file"
```

Comment:
It's working and not strictly necessary but I think I'd like to see this instead: `${formatted_files}${file}` just to be safe.